build/wikibase/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+## **wikibase@3.0.1** (2024-10-04)
+
+### 🚀 Features
```

Comment:
@lorenjohnson I am just re-reviewing this and think this should be "Fixes" not "Features". "Features" imply additional functionality for me, which would be a minor version bump in semver. Due to this being only a patch version bump, I'd expect no features listed. In conventional commits, this would be basically everything but `feat:`. WDYT?